WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailNewHeaderView.swift

```diff
@@ -247,8 +263,8 @@ struct ReaderDetailNewHeaderView: View {
             headerRow
             if let postTitle = viewModel.postTitle {
                 Text(postTitle)
-                    .font(.title)
-                    .fontWeight(.bold)
+                    .font(Font(viewModel.displaySetting.font(with: .title1, weight: .bold)))
+                    .foregroundStyle(Color(viewModel.displaySetting.color.foreground))
```

Comment:
I believe the variable `primaryTextColor` could be used here:
```suggestion
                    .foregroundStyle(Color(primaryTextColor))
```

Reply:
Good catch! This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22851/commits/6356219fae7bb80e10102ccaaa783c6598c0d879.